metabase/metabase
# use onRenderError callback to handle chart rendering errors

### Description

Unifies how rendering errors inside visualizations are handled by calling `props.onRenderError`

### How to verify

Add `throw new Error('foo')` to `getCartesianChartOption` function and ensure it shows an error message.
The error message is `Error: A cross-origin error was thrown. React doesn't have access to the actual error object in development` ~because of missing CORS setup on the Clojure side which will be added further.~

### Checklist
